websites/management/commands/markdown_cleanup.py

```diff
@@ -0,0 +1,97 @@
+"""Replace baseurl-based links with resource_link shortcodes."""
+import os
+from contextlib import ExitStack
+
+from django.core.management import BaseCommand
+from django.core.management.base import CommandParser
+from django.db import transaction
+
+from websites.management.commands.markdown_cleaning.baseurl_rule import (
+    BaseurlReplacementRule,
+)
+from websites.management.commands.markdown_cleaning.cleaner import (
+    WebsiteContentMarkdownCleaner,
+)
+from websites.management.commands.markdown_cleaning.resource_file_rule import (
+    ResourceFileReplacementRule,
+)
+from websites.management.commands.util import progress_bar
+from websites.models import WebsiteContent
+
+
+class Command(BaseCommand):
```

Comment:
This command is mostly the same as the old `markdown_cleanup_baseurl.py`, except it has a new positional argument "alias" that selects a rule from the `Rules` class property.